content/contributing-terms.en.md

```diff
@@ -512,3 +512,21 @@ export const removeSharesButton = [
   },
 ];
 ```
+
+### Handling terminated service
+
+If the service provider terminates a service, the associated terms will become unavailable. To apply a service closure in Open Terms Archive you must:
+
+1. Remove declared `<terms type>` in the declarations file. Hereinafter, an example of a service declaration where `documents` key has been emptied of `<terms type>` tracked 
+```json
+{
+  "name": "<service name>",
+  "documents": {}
+}
+```
+
+2. Fill the associated history file, as it is [documented](#terms-declaration-history).
```

Comment:
What should the `validUntil` date be? 🙂

Reply:
I guess what's already written is enough.

> Both for terms and filters history, the expiration date is declared in a property `validUntil`. It should be the authored date and time of the last snapshot commit for which the declaration is still valid.